solr/core/src/test/org/apache/solr/cloud/TestCloudRecovery.java

```diff
@@ -64,7 +64,7 @@ public static void setupCluster() throws Exception {
 
   @Before
   public void beforeTest() throws Exception {
-    configureCluster(2)
+    configureCluster(4)
```

Comment:
Looks like `TestCloudRecovery` changes are unrelated?

Reply:
thanks, sorry for the confuse!